apps/hash-api/package.json

```diff
@@ -14,6 +14,7 @@
     "test": "jest"
   },
   "dependencies": {
+    "@apps/hash-graph": "0.0.0-private",
```

Comment:
I guess this is required to have a correct dependency graph?

Reply:
Yes indeed